client/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -90,26 +93,26 @@ angular.module('managerApp').controller('XdslAccessCtrl', class XdslAccessCtrl {
       Until the task is finished, the diagnostic we get from
       the API is IRRELEVANT and should not be displayed to the user.
     */
-    this.$rootScope.$on('accessDiagnosticDetails:launch', () => {
-      this.launchDiagnostic();
-    });
+    this.$rootScope.$on('accessDiagnosticDetails:launch', () => this.launchDiagnostic());
 
     this.$rootScope.$on('accessDiagnosticDetails:get', () => {
+      let launchDiagnostic;
```

Comment:
Not yours but do you think we can listen to / emit from`$scope` instead ?

Reply:
Good point, I'd thought about this too, but as expected, `$rootScope` catches the event where `$scope` doesn't.

The emitter is located here : https://github.com/ovh-ux/ovh-manager-telecom/blob/master/client/app/telecom/pack/xdsl/access/diagnostic/details/pack-xdl-access-diagnostic-details.controller.js#L18 => It is the `$rootScope` :).

The whole event broadcast logic should be improved so, let's tackle this in another task (Creating a JIRA)